content/updates/2017-08-08-max-gov-migration-instructions.md

```diff
@@ -0,0 +1,91 @@
+
+**If you use a cloud.gov account and already have a MAX account**, double check that you can log in successfully to MAX and have a Secure+ SMS 2-factor device paired with your account.
+
+**If you use a cloud.gov account to log in and don’t have a MAX account,** you need to create one. Once we switch, you’ll be required to use that account to login to cloud.gov and all associated services.
+
+Create a MAX account by going to login.max.gov and clicking the green Register Now button in the top right corner or your agency’s logo. Make sure to add a 2-factor device to your account. The rest of this page has more detailed instructions for creating accounts if you need them.
+
+### Log in to cloud.gov with MAX
+
+Once you have an account, go to [*https://login.fr.cloud.gov*](https://login.fr.cloud.gov) and click on MAX.gov. This will take you to the MAX login page.
+
+<img src="/img/login-max.png" width="358" height="358" />
+
+**If you see your agency’s logo**, click it to sign in.
+
+<img src="/img/login-federated.png" alt="the MAX login screen with federated login options highlighted" width="624" height="574" />
+
+**If you don’t see your agency’s logo** and have a PIV/CAC card and a working reader, use your PIV/CAC card to login.
+
+<img src="/img/login-pivcac.png" alt="the MAX login screen with the PIV/CAC login method highlighted" width="624" height="574" />
+
+**If you don’t see your agency’s logo and don’t have a working card reader,** login with your MAX.gov user ID and password. Make sure to check “Use MAX Secure+ SMS 2-Factor” when you login.
+
+<img src="/img/login-userid.png" alt="the MAX login screen with the user ID and password login method highlighted" width="624" height="574" />
+
+### What about contractors?
+
+Contractors with a federal agency email address (.gov or .mil) should be able to self-register just like federal employees. If there are contractors or others on your team without a government email address, you’ll need to [*contact OMB MAX directly*](https://max.gov/maxportal/home.action) to get accounts for them.
+
+## Prepare for the switch
+
+We’re currently testing MAX login on our staging environment. In September 2017, we plan to switch all cloud.gov accounts to MAX.
+
+**Create a MAX account and make sure you have a 2-factor device paired with it.**
+
+You can create your account anytime and it only takes a few minutes. We encourage you to create your account now so you’re prepared well in advance of the migration. Once we make the switch, login with your MAX account and your account, apps, spaces, and orgs should appear as normal.
+
+When we switch in September you won’t be able to login with your cloud.gov account. If you don’t make a MAX account before then, your apps, orgs, and spaces will not go away but you won’t be able to login to either the dashboard or the CLI. Once you create a MAX account, you’ll regain access.
```

Comment:
I think we may need a statement earlier in the post that if you make the MAX.gov account with the same email address you used for your cloud.gov account, you'll still have access to the same orgs/spaces/apps (if that's right).